app/views/layouts/application.html.haml

```diff
@@ -52,6 +52,10 @@
                 - current_site.posts.published.order_by_recently.limit(5).each do |post|
                   = render partial: 'posts/summary_with_content', locals: {post: post}
 
+            - if current_site.serials.present?
+              %h2.subheading.subheading--aside.subheading--press-articles 連載
+              = link_to 'すべての連載を見る', serials_path(current_site)
```

Comment:
BTW, `serials_path` and `serials_url` couldn't be used in production.
see: https://github.com/bm-sms/daimon-news/blob/serials-contents/config/routes.rb#L37-L39

Can you remove this guard?